CONTRIBUTING.md

```diff
@@ -91,6 +91,10 @@ Make sure you have [Gradle](http://gradle.org) installed, as
 Elasticsearch uses it as its build system. Gradle must be at least
 version 3.3 in order to build successfully.
 
+We support development in the Eclipse and IntelliJ IDEs. For Eclipse, the
```

Comment:
I wonder if we should go as far as to say we "support" this, since we didn't have any promise attached to it so far. Is "suggest" too vague here? And saying that these are the minimum versions we test with ourselves? Just my 2 cent.